### PR TITLE
Update Zola to 0.14.1 (fixes #758)

### LINKED ISSUE #758: Site fails to build: Variable `config.default_language` not found in context while rendering 'index.html'
See https://github.com/getzola/zola/issues/1442#issuecomment-912368126

Basically I get this error when trying to build with Zola 0.14.1 on both Windows 10 and WSL2:
```
Error: Failed to render section '[...]\rust-gamedev.github.io\content\_index.md'
Reason: Failed to render 'index.html'
Reason: Variable `config.default_language` not found in context while rendering 'index.html'
```

Even though the "default_language" is clearly present in config.toml:
```
title = "Rust GameDev WG"
description = "Stay up to date with the progress and recent developments in the Rust Game Development Working Group."
base_url = "https://gamedev.rs/"
default_language = "en"
compile_sass = true
highlight_code = true
generate_feed = true
feed_filename = "rss.xml"
...
```

I'm not sure if this is a Zola issue, or an issue with the newsletter, so I'm opening it on both issue trackers.

Update: changing `config.default_language` to `lang` in index.html seems to fixes the issue.

### PR DESCRIPTION
**Pull request by 17cupsofcoffee**

### CODE REVIEW

**Comment by 17cupsofcoffee** [comment]
I switched the `text`/`plain` language tags to `txt`, as Zola only recognizes the latter as a valid language (https://github.com/getzola/zola/issues/1633). Wouldn't have broken anything if we left it as is, but did log a lot of warnings.

**Review by AngelOnFira — APPROVED** [review]
Great, looks good 👍

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The site is built and deployed automatically from the repo (see .github/workflow
 
 To preview/experiment locally:
 
-1) [Install Zola][zola-get].
```
Comment (ozkriff) [inline README.md:15]: maybe it's better to link ci.yml here for future-proofing?

**Review by ozkriff — APPROVED** [review]
:rocket:

File: README.md (hunk 1 above)
Comment (17cupsofcoffee) [reply README.md:15]: I think it's worth having the version number in the README so people don't have to read through the CI config, but I think it'd be good to link it here too :)

**Pull request merged**